neqo-bin/src/lib.rs

```diff
@@ -24,6 +24,11 @@ pub mod client;
 pub mod server;
 pub mod udp;
 
+/// Firefox default value
+///
+/// See `network.buffer.cache.size` pref <https://searchfox.org/mozilla-central/rev/f6e3b81aac49e602f06c204f9278da30993cdc8a/modules/libpref/init/all.js#3212>
+const STREAM_IO_BUFFER_SIZE: usize = 32 * 1024;
```

Comment:
32k < MTU, so maybe this is hitting an issue there?

Reply:
Thanks for taking a look! I believe this is fully resolved by https://github.com/mozilla/neqo/pull/2035. Benchmark regressions are gone. Download benchmark shows a 25% improvement.